profile/src/v100/entity/persona/persona_data/identified_entry.rs

```diff
@@ -0,0 +1,154 @@
+use std::str::FromStr;
+
+use derive_more::Display;
+use serde::{Deserialize, Serialize};
+use uuid::Uuid;
+
+use crate::HasPlaceholder;
+
+use super::{Name, Variant};
+
+type PersonaDataEntryID = Uuid;
+
+// TODO: Needs to be made generic when adding more entry_kinds. Right now uniffi::Record complains if the function is generic.
```

Comment:
I will try to give you a generic one, should be doable I think, since IdentifiedVecVia<T> is generic and I've exposed it to UniFFI